django_webapps/settings_sample.py

```diff
@@ -125,3 +130,19 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_SCHEMA_CLASS": "api.schema.CustomSchema",
+}
+
+SPECTACULAR_SETTINGS = {
+    "SERVE_INCLUDE_SCHEMA": False,
+}
+
+#  If you import something from rest_framework before REST_FRAMEWORK setting,
+#  this setting will be ignored.
+from rest_framework import viewsets  # noqa
+
+# This is needed to be able to use annotations like "viewsets.ReadOnlyModelViewSet[Teacher]"
+# for mypy to stop ****ing our brains.
+django_stubs_ext.monkeypatch(extra_classes=[viewsets.ModelViewSet, viewsets.ReadOnlyModelViewSet])
```

Comment:
```suggestion

REST_FRAMEWORK = {
    "DEFAULT_SCHEMA_CLASS": "api.schema.CustomSchema",
}

SPECTACULAR_SETTINGS = {
    "SERVE_INCLUDE_SCHEMA": False,
}


# The following is needed to be able to use annotations like "viewsets.ReadOnlyModelViewSet[Teacher]"
# for mypy to stop ****ing our brains.

#  If you import something from rest_framework before REST_FRAMEWORK setting,
#  this setting will be ignored.
from rest_framework import viewsets  # noqa
django_stubs_ext.monkeypatch(extra_classes=[viewsets.ModelViewSet, viewsets.ReadOnlyModelViewSet])
```

Reply:
I just think it should be absolutely clear that this import is only needed for mypy